Sources/Plasma/FeatureLib/pfPython/pyGUIControlListBox.cpp

```diff
@@ -75,91 +75,33 @@ class pfColorListElement : public pfGUIListText
             kSelectUseGUIColor,
         };
 
-        pfColorListElement( const char *string1, hsColorRGBA color1, const char *string2, hsColorRGBA color2, uint32_t inheritalpha, int32_t fontsize=-1 )
+        pfColorListElement( ST::string string1, hsColorRGBA color1, ST::string string2, hsColorRGBA color2, uint32_t inheritalpha, int32_t fontsize=-1 )
         {
-            if ( string1 )
-            {
-                fString1 = hsStringToWString(string1);
-                fText = ST::string();
-            }
-            else
-            {
-                fString1 = nullptr;
-                fText = ST::string();
-            }
+            fText = ST::string();
+            fString1 = std::move(string1);
             fTextColor1 = color1;
-            if (string2)
-                fString2 = hsStringToWString(string2);
-            else
-                fString2 = nullptr;
+            fString2 = std::move(string2);
             fTextColor2 = color2;
             fInheritAlpha = inheritalpha;
             fJustify = kLeftJustify;
             fOverrideFontSize = fontsize;
         }
 
-        pfColorListElement( const wchar_t *string1, hsColorRGBA color1, const wchar_t *string2, hsColorRGBA color2, uint32_t inheritalpha, int32_t fontsize=-1 )
+        pfColorListElement(ST::string string1, hsColorRGBA color1, uint32_t inheritalpha, int32_t fontsize = -1)
```

Comment:
Did you intend to murder `color2`?

Reply:
Yes, I converted this 2nd constructor (`wchar_t*` vs `char*`) into a 1-color constructor for usage by AddTextWColorW and AddTextWColorWSizeW in the glue methods below. Previously, it was just passing nullptr and a default-constructed `hsColorRGBA` object in because those had no second string or color. And, of course, when I made all of these `ST::string`, there's no way to pass in nullptr... nor did I really see a need to. I'm default-constructing string2 and color2 below.